src/main/java/frc/robot/hardware/request/phoenix6/Phoenix6RequestBuilder.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.hardware.request.phoenix6;
+
+import com.ctre.phoenix6.controls.PositionVoltage;
+import com.ctre.phoenix6.controls.TorqueCurrentFOC;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.controls.VoltageOut;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class Phoenix6RequestBuilder {
+
+	public static Phoenix6Request<Rotation2d> build(PositionVoltage positionVoltage) {
+		return new Phoenix6Request<>(new Rotation2d(), positionVoltage, setPoint -> positionVoltage.withPosition(setPoint.getRotations()));
```

Comment:
why do you create a default set point in the builder here and get it as parameter in spark max builder